### PR TITLE
prep 2-8-0

### PR DESCRIPTION
**Pull request by MattHJensen**
Prepare Release 2.8.0. 

@Peter-Metz, could you please look this over?

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ Go [here](https://github.com/PSLmodels/Tax-Calculator/pulls?q=is%3Apr+is%3Aclose
 for a complete commit history.
 
 
+2020-02-28 Release 2.8.0
+------------------------
+(last merged pull request is
+[#2400](https://github.com/PSLmodels/Tax-Calculator/pull/2400))
```
Comment (Peter-Metz) [inline RELEASES.md]: I think this should be #2399

Comment (MattHJensen) [reply RELEASES.md]: I put the PR of the very last merged PR, which is this one.

**Comment by Peter-Metz** [comment]
@MattHJensen looks good to me!

**Comment by MattHJensen** [comment]
@Peter-Metz, thanks for the review. Merging and releasing.

**Pull request merged**